packages/database/drizzle/0036_burly_cargill.sql

```
@@ -0,0 +1,20 @@
+-- Full-text search (tsvector + ts_rank) and fuzzy matching (pg_trgm).
+-- NOTE: hand-edited after `drizzle-kit generate` — the GENERATED expressions use bare column
+-- names (Postgres rejects table-qualified references in generation expressions) and the
+-- extension + GIN/pg_trgm indexes are added here (drizzle-kit cannot serialise the gin_trgm_ops
+-- operator class). The 0031 snapshot intentionally does not track these indexes.
+-- Enum columns (macros.language, protocols.family) are matched at query time, not in the stored
+-- vector, because enum->text casts are not immutable and cannot live in a generated column.
+CREATE EXTENSION IF NOT EXISTS pg_trgm;--> statement-breakpoint
+ALTER TABLE "experiments" ADD COLUMN "search_vector" tsvector GENERATED ALWAYS AS (setweight(to_tsvector('english', coalesce("name", '')), 'A') || setweight(to_tsvector('english', coalesce("description", '')), 'B')) STORED;--> statement-breakpoint
+ALTER TABLE "macros" ADD COLUMN "search_vector" tsvector GENERATED ALWAYS AS (setweight(to_tsvector('english', coalesce("name", '')), 'A') || setweight(to_tsvector('english', coalesce("description", '')), 'B')) STORED;--> statement-breakpoint
+ALTER TABLE "protocols" ADD COLUMN "search_vector" tsvector GENERATED ALWAYS AS (setweight(to_tsvector('english', coalesce("name", '')), 'A') || setweight(to_tsvector('english', coalesce("description", '')), 'B')) STORED;--> statement-breakpoint
+CREATE INDEX "experiments_search_vector_idx" ON "experiments" USING gin ("search_vector");--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)